examples/multisig/README.md

```diff
@@ -0,0 +1,59 @@
+# Multisignature delegated account support
+
+Examples to demonstrate authorizing transactions based on logic signature signed by multiple accounts.
+
+Two examples are provided
```

Comment:
Please be more precise and explain exactly what the examples is doing. A developer, who will look at this file will not know what the smart-contract is checking nor what the script is doing. Please describe that.

Reply:
Added a description in `readme.md`. Let me know.